### PR TITLE
Dashboard: swisstopo showcase projects (permalinks collection, no project geometries)

### PR DESCRIPTION
**Pull request by vladyslav-tk**

### CODE REVIEW

**Comment by eleu** [comment]
really great to see that!

Two small comments:
* Font size of "Description" needs adaption
* Can we activate the target point for some of the links? (maybe means adding it to the permalink, actually)

File: src/constants.ts [hunk 1]
```diff
@@ -105,3 +105,54 @@ export const SHORTLINK_HOST_BY_PAGE_HOST = {
   'test.viewer.swissgeol.ch': 'link.swissgeol.ch',
   'viewer.swissgeol.ch': 'link.swissgeol.ch',
 };
+
+export const SHOWCASE_PROJECTS = [
```
Comment (gberaudo) [inline src/constants.ts:109]: I thinks this should be put in a static file and fetched when the dashboard is opened for the first time.

**Review by gberaudo — APPROVED** [review]
LGTM

File: src/constants.ts (hunk 1 above)
Comment (vladyslav-tk) [reply src/constants.ts]: updated

**Pull request merged**